x-pack/test/functional/services/data_stream.ts

```diff
@@ -23,6 +23,11 @@ export function DataStreamProvider({ getService, getPageObject }: FtrProviderCon
     deleteOriginal: false,
   };
 
+  async function existsDataStream(stream: string) {
```

Comment:
Where is this function used? I don't see a test case that uses it. Also, wouldn't it be better to call it something like `getDataStream`?

Reply:
It is not used anywhere yet, but initially I thought to expose it to handle errors when creating a data stream with the same name of an existing one (i.e. manual stop of a failing FTR and subsequent restart), then I've addressed the issue with a bespoken code here: https://github.com/elastic/kibana/pull/200007/files#diff-460cf76ae380343d7b854f412b3a2daa5ccde74135d55c7b383d5d1321b15661R261-R266

But I've left this method exposed as I thought it may be useful to have a specific utility to check the existence of a datastream. If you want I can remove it, no strong opinion.

Reply:
I see. I would say let's remove it for now as it's not needed yet. Also, it's technically a single line of code so not sure if we need a helper function for that.

Reply:
Removed with [6c0cf8e](https://github.com/elastic/kibana/pull/200007/commits/6c0cf8e19422523d2abd49b3fdd1b64cbed61a54)